.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,4 +23,9 @@ Closes #
 
 #### Do we need any specific form for testing your changes? If so, please attach one.
 
-#### Does this change require updates to documentation? If so, please file an issue at https://github.com/opendatakit/docs/issues/new and include the link below.
\ No newline at end of file
+#### Does this change require updates to documentation? If so, please file an issue at [Here]( https://github.com/opendatakit/docs/issues/new) and include the link below.
+
+#### Before submitting this PR, please make sure you have tried:
+- [ ] Run the tests with `./gradlew pmd checkstyle lint findbugs` to make sure you didn't break anything.
+
+- [ ] If you have multiple commits with the same kind of changes, please combine them into one commit by squashing them.
```

Comment:
GitHub now makes it really easy for reviewers to squash commits on merge which also keeps a linear history so I'd tend to leave this out as it can be a barrier to new contributors and can make review harder!

Reply:
I think some commits like:
`clean code1`
`clean code2`
`clean code3`
can be squashed into one.
> it can be a barrier to new contributors and can make review harder
Yes indeed, can't we set this not a compulsory option or put this in `CONTRIBUTION_GUIDE.md` ?